Global.gd

```diff
@@ -20,14 +24,34 @@ var TEXT_COLOR = {
 	"HEAL": "2eff27",
 }
 
-const PLAYER_NAME = "Astronaut"
 # A list of scenes that are persisted, default null for each
 var persisted_scenes = [null]
 var previous_scene = null
 var current_scene = null
+
+# game mechanics
+const PLAYER_NAME = "Astronaut"
+const BASE_HEALTH = 100
+const BASE_OXYGEN = 30
+var player = null
+var current_level = 1
+
 func _ready():
 	var root = get_tree().get_root()
 	current_scene = root.get_child(root.get_child_count() - 1)
+	player = build_player()
+
+func build_player():
+	# TODO: Add upgrades
+	var player_stats = Creature.Stats.new(1, 1, 1, 1, 1)
```

Comment:
magic numbers, please make dictionary with description of values. Will be helpful for balancing when we want to modify the starting array